library/extractor/src/main/java/com/google/android/exoplayer2/extractor/ogg/DefaultOggSeeker.java

```diff
@@ -69,6 +69,7 @@
    */
   public DefaultOggSeeker(
       StreamReader streamReader,
+      long streamSerialNumber,
```

Comment:
We won't need to change the signature of this function if we can make streamSerialNumber as public member of StreamReader class.